[Android] Using deprecated API

## 🐛 Bug Report

When compiling my app I had this warning:
```
Note: /home/colas/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_tts-0.2.6/android/src/main/java/com/tundralabs/fluttertts/FlutterTtsPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```
I added this in the ```build.gradle``` file to have the details:
```gradle
allprojects {
    tasks.withType(JavaCompile) {
        options.compilerArgs << "-Xlint:deprecation"
    }
}
```
And I got this result:
```
/home/colas/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_tts-0.2.6/android/src/main/java/com/tundralabs/fluttertts/FlutterTtsPlugin.java:77: warning: [deprecation] getDefaultLanguage() in TextToSpeech has been deprecated
                      : tts.getDefaultLanguage();
                           ^
1 warning
```
Looking at the API reference for [```getDefaultLanguage()```](https://developer.android.com/reference/android/speech/tts/TextToSpeech.html#getDefaultLanguage()) it says that it's deprecated since API level 21.

However in ```build.gradle``` we do specify:
```gradle
minSdkVersion 21
```

That's why I think that the following ternary should be replaced:
```java
Locale locale = Build.VERSION.SDK_INT >= Build.VERSION_CODES.M
                      ? tts.getDefaultVoice().getLocale()
                      : tts.getDefaultLanguage();
```

This gets the job done without any warning and is valid since our ```minSdkVersion``` is 21:
```java
Locale locale = tts.getDefaultVoice().getLocale();
```

### Expected behavior

No warning.

### Reproduction steps

Building a project using the flutter_tts plugin for Android.

### Configuration

**Version:** 0.2.6

**Platform:** 
- [ ] :iphone: iOS
- [x] :robot: Android

--- a/android/src/main/java/com/tundralabs/fluttertts/FlutterTtsPlugin.java
+++ b/android/src/main/java/com/tundralabs/fluttertts/FlutterTtsPlugin.java
@@ -72,9 +72,7 @@ public class FlutterTtsPlugin implements MethodCallHandler {
             invokeMethod("tts.init", true);
 
             try {
-              Locale locale = Build.VERSION.SDK_INT >= Build.VERSION_CODES.M
-                      ? tts.getDefaultVoice().getLocale()
-                      : tts.getDefaultLanguage();
+              Locale locale = tts.getDefaultVoice().getLocale();
               if (isLanguageAvailable(locale)) {
                 tts.setLanguage(locale);
               }

